client/blocks/importer/hooks/use-site-can-migrate.ts

```diff
@@ -1,60 +0,0 @@
-import { useEffect, useRef, useState } from 'react';
-import { useMigrationEnabledInfoQuery } from 'calypso/data/site-migration/use-migration-enabled';
```

Comment:
This hook doesn't seem to be used anymore.

Reply:
Thank you for flagging! I removed it too, although it was not enough to reach 5k deletions 😅